scripts/compare_gas.py

```
@@ -0,0 +1,69 @@
+    try:
+        with open(file_path, 'r') as f:
+            content = f.read()
+    except FileNotFoundError:
+        raise FileNotFoundError(f"File not found: {file_path}")
+
+    match = re.search(pattern, content, re.MULTILINE)
+    if not match:
+        raise ValueError(f"Pattern not found in {file_path}: {pattern}")
+    try:
+        return int(match.group(1))
+    except (IndexError, ValueError) as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)